crates/re_data_source/src/data_loader/loader_external.rs

```diff
@@ -0,0 +1,188 @@
+use std::io::Read;
```

Comment:
You should document somewhere what we expect from a loader, eg in a `//!`-level docstring

In particular:
* what arguments to expect (`--recording-id`)
* what exit code to use for unsupported file types
* what and where to output (.rrd data to `stdout`)

This should also be added as a how-to guide on rerun.io/docs!

Reply:
I have an upcoming PR that adds a web guide, I'll add a link to that guide in the doc string then